tasks/patternpack.js

```diff
@@ -71,8 +71,7 @@ module.exports = function (grunt) {
     return _.defaultsDeep(_.cloneDeep(overrideValue), value);
   }
 
-  function setupOptions(context) {
-    var path = require("path");
+  function getOptions(context) {
```

Comment:
`setupOptions()` has been split into 2 parts.  The first part `getOptions()` is responsible for getting the options and applying default/override values.  The second part `transformOptions()` is responsible for mapping paths from the current directory to the `node_modules/patternpack` directory.